### PR TITLE
Fix alignment of sponsors columns + column contents

### PR DESCRIPTION
**Pull request by stefanpenner**
before:

screen shot 2016-07-26 at 11 16 02 am

after:

screen shot 2016-07-26 at 11 16 12 am

### CODE REVIEW

**Comment by stefanpenner** [comment]
`/sponsors` + `/ember-users` both appear ok

**Comment by stefanpenner** [comment]
The existing CSS appears to have existed for a reason, but does not appear to work as originally intended. Removing it appears to have resulted in a fix, but no obvious regression. If there is something I'm missing, please share what should happen and I can look at as well.

**Comment by chancancode** [comment]
@stefanpenner I think the goal is to use the full width of the page, i.e. have the left side of the rectangle aligned with the left side of the header/text.

File: source/stylesheets/site.css.scss [hunk 1]
```diff
@@ -1579,33 +1579,18 @@ ul.showcase {
 
   width: 33.333%;
   margin: 10px 0;
-  &:nth-child(3n+1) {
-    a {
-      margin: 0 auto 0 0;
-    }
-  }
```
Comment (chancancode) [inline source/stylesheets/site.css.scss]: Changing these to:

```css
&:nth-child(3n+1) {
  > * {
    display: block;
    margin: 0 auto 0 0;
    width: 210px;
  }		
}
```

...and so on, seems to more or less satisfy that goal

Comment (chancancode) [inline source/stylesheets/site.css.scss]: result

Comment (chancancode) [inline source/stylesheets/site.css.scss]: If you want to go with this, you would also need to target `> *` instead of `a` for other screen sizes

Comment (stefanpenner) [inline source/stylesheets/site.css.scss]: If it's up to me, I'm ok with losing left-hand and right-hand margin alignment, as it is still pleasantly centered. It is possible to accomplish the alignment, but Im not terribly motivated to, so if someone is...

**Pull request closed without merging**